CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps or point out the part of Atom which the suggestion is related to. You can use [this tool](http://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
+* **Explain why this enhancement would be useful** to most Lisk and Lisk-JS users.
+* **List some other text editors or applications where this enhancement exists.**
+* **Specify which version of Lisk and Lisk-JS you're using.**
+* **Specify the name and version of the OS you're using.**
+
+### Pull Requests
+
+* Include issue numbers in the PR title, at the end with: ```Description - Closes #IssueNumber```
+* Include screenshots and animated GIFs in your pull request whenever possible.
+* Follow the [JavaScript](https://github.com/airbnb/javascript) styleguide from airbnb.
+* Document new code based on the [YuiDoc Documentation Styleguide](http://yui.github.io/yuidoc/)
+* End all files with a newline
+* Every new code must come with complete test coverage
+* Update the Readme for the changes that adhere to your new code
+
+## Styleguides
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+* Limit the first line to 72 characters or less
+* Reference issues and pull requests liberally after the first line
+* Consider starting the commit message with an applicable emoji:
+    * :art: `:art:` when improving the format/structure of the code
+    * :racehorse: `:racehorse:` when improving performance
+    * :non-potable_water: `:non-potable_water:` when plugging memory leaks
+    * :memo: `:memo:` when writing docs
+    * :penguin: `:penguin:` when fixing something on Linux
+    * :apple: `:apple:` when fixing something on macOS
+    * :checkered_flag: `:checkered_flag:` when fixing something on Windows
+    * :bug: `:bug:` when fixing a bug
+    * :fire: `:fire:` when removing code or files
+    * :green_heart: `:green_heart:` when fixing the CI build
+    * :white_check_mark: `:white_check_mark:` when adding tests
+    * :lock: `:lock:` when dealing with security
+    * :arrow_up: `:arrow_up:` when upgrading dependencies
+    * :arrow_down: `:arrow_down:` when downgrading dependencies
+    * :shirt: `:shirt:` when removing linter warnings
+
+### JavaScript Styleguide
+
+All JavaScript must adhere to [JavaScript Standard Style](http://standardjs.com/).
```

Comment:
How does https://www.npmjs.com/package/eslint-config-lisk-base factor into this?

Reply:
We currently don't use JavaScript Standard Style.